client/components/team/TeamEditor.jsx

```diff
@@ -131,6 +186,10 @@ TeamEditor = class TeamEditor extends Component {
     this.setState({ [name]: value });
   }
 
+  _toggleInPerson(enableInPerson) {
```

Comment:
nitpick: Would this function be better named _setInPerson(value)? Since it's being passed the desired value it's not really toggling.

Reply:
great nit. absolutely true

Reply:
fixed